truffle-config.js

```diff
@@ -1,11 +1,20 @@
 const Web3 = require('web3')
-
-const LocalWallet = require('./libs/celo/packages/contractkit/lib/wallets/local-wallet').LocalWallet
-const CeloProvider = require('./libs/celo/packages/contractkit/lib/providers/celo-provider').CeloProvider
-const generateUtils = require('./libs/celo/packages/celotool/lib/lib/generate_utils')
+const bip32 = require('bip32')
```

Comment:
tl;dr changes to this file?

Reply:
These are artefacts of removing `celo-monorepo` as a submodule which weren't treated at the time.